### PR TITLE
Avatar badge

### PR DESCRIPTION
**Pull request by sakshi006**

### CODE REVIEW

**Comment by sakshi006** [comment]
Link to the website : https://frosty-meninsky-3ec285.netlify.app/

File: components/Avatar/Avatar.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
+.img {
+  width: 3.6rem;
+  height: 3.8rem;
+  border: 1.1px solid black;
+  margin: 2rem 1.5rem;
+}
```
Comment (pnchinmay) [inline components/Avatar/Avatar.css:7]:
```suggestion
    object-fit: cover;
}
```
Refer this:
[MDN Object Fit](https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit)
Also, refer this article by freecodecamp.
https://www.freecodecamp.org/news/css-responsive-image-tutorial/

Comment (pnchinmay) [reply components/Avatar/Avatar.css:6]: Refer this:
[MDN Object Fit](https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit)

**Review by pnchinmay — CHANGES_REQUESTED** [review]
Suggested Changes: 
- Responsive Image

File: components/Badges/Badges.css [hunk 2]
```diff
@@ -77,8 +78,8 @@ iframe{
 }
 
 .img {
-    width: 3rem;
-    height: 3.2rem;
+    width: 4rem;
+    height: 4.2rem;
     border: 1.1px solid black;
     margin: -1rem 1.5rem;
```
Comment (pnchinmay) [inline components/Badges/Badges.css:84]: Same property needed here for responsive images

**Review by pnchinmay — CHANGES_REQUESTED** [review]

**Pull request merged**